extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -123,3 +124,47 @@ export async function createAzureFunction(connectionString: string, schema: stri
 		azureFunctionUtils.overwriteAzureFunctionMethodBody(functionFile);
 	}
 }
+
+/**
+ * Adds a SQL Binding to a specified Azure function in a file
+ * @param bindingType Type of SQL Binding
+ * @param filePath Path of the file where the Azure Functions are
+ * @param functionName Name of the function where the SQL Binding is to be added
+ * @param objectName Name of Object for the SQL Query
+ * @param connectionStringSetting Setting for the connection string
+ * @returns
+ */
+export async function addSqlBinding(
```

Comment:
You're returning a Promise wrapped in a Thenable which doesn't seem correct - it should just be a promise